src/st/class-wpml-pb-integration.php

```diff
@@ -68,6 +68,14 @@ public function resave_post_translation_in_shutdown( WPML_Post_Element $post_ele
 
 		$updated_packages = $this->factory->get_package_strings_resave()->from_element( $post_element );
 
+		if ( ! $updated_packages ) {
```

Comment:
@strategio I would be tempted to add another hook to `WPML_PB_Handle_Post_Body` called `wpml_pb_copy_body_if_no_packages` that calls the copy function. I'll leave it to you to decide.